### PR TITLE
Update to Org Contacts in beta

### PR DESCRIPTION
**Pull request by dkershaw10**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8cf2218](https://github.com/dkershaw10/microsoft-graph-docs/pull/3609/commits/8cf2218fb7b33386f776b82bd83cf49413c941be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/orgcontact-get.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/api/orgcontact-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/orgcontact-list.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/api/orgcontact-list.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/orgcontact.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/resources/orgcontact.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/physicalofficeaddress.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/resources/physicalofficeaddress.md) | :white_check_mark:Succeeded | |
[api-reference/beta/toc.yml](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C12%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812121156259379-3609%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=drg2Jo4bO8kFNO0jFdX61rUs0Xlsxl2xidJaR7w23mU%3D&st=2018-12-12T11%3A56%3A04Z&se=2019-01-12T12%3A01%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/orgcontact-get.md [hunk 1]
```diff
@@ -59,14 +59,27 @@ Content-type: application/json
...
+  "addresses":[
+      {
+        "city": "string",
+        "countryOrRegion": "string",
+        "officeLocation": "string",
+        "postalCode": "string",
+        "postOfficeBox": "string",
+        "state": "string",
+        "street": "string",
+        "type": "string"
```
Comment (eketo-msft) [inline api-reference/beta/api/orgcontact-get.md]: Copy paste?

File: api-reference/beta/api/orgcontact-get.md [hunk 2]
```diff
@@ -59,14 +59,27 @@ Content-type: application/json
...
 {
-  "businessPhones": [
-    "businessPhones-value"
+  "addresses":[
+      {
+        "city": "string",
+        "countryOrRegion": "string",
+        "officeLocation": "string",
+        "postalCode": "string",
+        "postOfficeBox": "string",
```
Comment (eketo-msft) [inline api-reference/beta/api/orgcontact-get.md]: We don't have POBox in the API, did we miss this in API review?

File: api-reference/beta/resources/orgcontact.md [hunk 3]
```diff
@@ -23,73 +65,23 @@ Here is a JSON representation of the resource
...
   "mail": "string",
   "mailNickname": "string",
-  "mobilePhone": "string",
-  "officeLocation": "string",
-  "onPremisesLastSyncDateTime": "String (timestamp)",
+  "onPremisesLastSyncDateTime": "string (timestamp)",
   "onPremisesProvisioningErrors": [{"@odata.type": "microsoft.graph.onPremisesProvisioningError"}],
   "onPremisesSyncEnabled": true,
-  "postalCode": "string",
+  "phones": [{"@odata.type": "microsoft.graph.phones"}],
```
Comment (eketo-msft) [inline api-reference/beta/resources/orgcontact.md]: type should be microsoft.graph.phone.

File: api-reference/beta/resources/orgcontact.md [hunk 4]
```diff
@@ -7,6 +7,48 @@ description: "Here is a JSON representation of the resource"
...
+| onPremisesProvisioningErrors |[onPremisesProvisioningError](onpremisesprovisioningerror.md) collection       | List of any synchronization provisioning errors for this organizational contact.                                                                                                                                                                                                                                                                                                |
+|onPremisesSyncEnabled|Boolean|**true** if this object is synced from an on-premises directory; **false** if this object was originally synced from an on-premises directory but is no longer synced and now mastered in Exchange; **null** if this object has never been synced from an on-premises directory (default).|
+| phones                       | [phone](phone.md)                             | List of phones for this organizational contact. Phone types can be mobile, business, and businessFax. Only one of each type can ever be present in the collection.                                                                                                                       |
```
Comment (eketo-msft) [inline api-reference/beta/resources/orgcontact.md]: This should be phone collection.

File: api-reference/beta/resources/physicalofficeaddress.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title: "physicalOfficeAddress resource type"
+description: "Represents the business address of a resource such as a contact or event."
+---
+
+# physicalOfficeAddress resource type
+
+Represents the business address of a resource such as an organizational contact.  Derives from the [physicalAddress](physicalAddress.md) resource type.
```
Comment (eketo-msft) [inline api-reference/beta/resources/physicalofficeaddress.md]: I don't think we are deriving this.  Is this where the two extra properties come from (type and postOfficeBox)?

File: api-reference/beta/api/orgcontact-get.md (hunk 2 above)
Comment (dkershaw10) [reply api-reference/beta/api/orgcontact-get.md]: Yeah - let's discuss offline

File: api-reference/beta/resources/orgcontact.md (hunk 4 above)
Comment (dkershaw10) [reply api-reference/beta/resources/orgcontact.md]: Nice - will fix

File: api-reference/beta/resources/orgcontact.md (hunk 3 above)
Comment (dkershaw10) [reply api-reference/beta/resources/orgcontact.md]: Good catch

File: api-reference/beta/resources/physicalofficeaddress.md (hunk 5 above)
Comment (dkershaw10) [reply api-reference/beta/resources/physicalofficeaddress.md]: Yes.  That might be my confusion :)

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[1e5a059](https://github.com/dkershaw10/microsoft-graph-docs/pull/3609/commits/1e5a0592f913223f9a2fe33cc5c5fe61e405393b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/orgcontact-get.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/api/orgcontact-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/orgcontact-list.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/api/orgcontact-list.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/orgcontact.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/resources/orgcontact.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/physicalofficeaddress.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/resources/physicalofficeaddress.md) | :white_check_mark:Succeeded | |
[api-reference/beta/toc.yml](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/dkershaw_orgContacts/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812132054484078-3609%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=otWhk2jFRn4ELykPbHyOAfdd413OmN40UrcfKy4UJUk%3D&st=2018-12-13T20%3A54%3A36Z&se=2019-01-13T20%3A59%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**